File: SWB4/Client/OpenOffice/OfficeComunicationInterfaces/src/org/semanticwb/office/interfaces/CalendarInfo.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.semanticwb.office.interfaces;

/**
 *
 * @author victor.lorenzana
 */
public class CalendarInfo {
    public String id;
    public String xml;
    public boolean active;
    public String title;
    @Override
    public boolean equals(Object obj)
    {
        if (obj == null)
        {
            return false;
        }
        if (getClass() != obj.getClass())
        {
            return false;
        }
        final CalendarInfo other = (CalendarInfo) obj;
        if ((this.id == null) ? (other.id != null) : !this.id.equals(other.id))
        {
            return false;
        }
        return true;
    }

    @Override
    public int hashCode()
    {
        int hash = 7;
        hash = 89 * hash + (this.id != null ? this.id.hashCode() : 0);
        return hash;
    }

    @Override
    public String toString()
    {
        return title.toString();
    }


}

File: SWB4/Client/Microsoft Office/INFOTEC WebBuilder 4/WBOffice4/Interfaces/CalendarInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WBOffice4.Interfaces
{
    public class CalendarInfo
    {
        public String id;
        public String xml;
        public bool active;
        public String title;

        public override bool Equals(Object obj)
        {
            if (obj == null)
            {
                return false;
            }
            if (GetType() != obj.GetType())
            {
                return false;
            }
            CalendarInfo other = (CalendarInfo)obj;
            if ((this.id == null) ? (other.id != null) : !this.id.Equals(other.id))
            {
                return false;
            }
            return true;
        }


        public override int GetHashCode()
        {
            int hash = 7;
            hash = 89 * hash + (this.id != null ? this.id.GetHashCode() : 0);
            return hash;
        }


        public override String ToString()
        {
            return title.ToString();
        }
    }
}
